src/python/aim/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 from .record import Record
 from .sequence import Sequence
-from .container import Container
+from .container import Container, Property
```

Comment:
why won't include the `Property` in `__all__` as well?

Reply:
good catch. will make changes soon